data/typelibrary/utils-1.0.0/typelib/assembling/BYTES_TO_DWORD.fct

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Function Name="BYTES_TO_DWORD" Comment="">
+	<Identification Standard="61499-1" Description="Copyright (c) 2024 HR Agrartechnik GmbH    &#10;    &#10;This program and the accompanying materials are made     &#10;available under the terms of the Eclipse Public License 2.0     &#10;which is available at https://www.eclipse.org/legal/epl-2.0/     &#10;     &#10;SPDX-License-Identifier: EPL-2.0" >
+	</Identification>
+	<VersionInfo Version="1.0" Author="franz" Date="2024-02-22">
+	</VersionInfo>
+	<CompilerInfo>
+	</CompilerInfo>
+	<InterfaceList>
+		<EventInputs>
+			<Event Name="REQ" Type="Event" Comment="">
+				<With Var="IN_00"/>
+				<With Var="IN_01"/>
+				<With Var="IN_02"/>
+				<With Var="IN_03"/>
+			</Event>
+		</EventInputs>
+		<EventOutputs>
+			<Event Name="CNF" Type="Event" Comment="">
+				<With Var=""/>
+			</Event>
+		</EventOutputs>
+		<InputVars>
+			<VarDeclaration Name="IN_00" Type="BYTE" Comment="BYTE 00"/>
+			<VarDeclaration Name="IN_01" Type="BYTE" Comment="BYTE 01"/>
+			<VarDeclaration Name="IN_02" Type="BYTE" Comment="BYTE 02"/>
+			<VarDeclaration Name="IN_03" Type="BYTE" Comment="BYTE 03"/>
+		</InputVars>
+		<OutputVars>
+			<VarDeclaration Name="" Type="DWORD" Comment=""/>
+		</OutputVars>
+	</InterfaceList>
+	<FunctionBody>
+		<ST><![CDATA[FUNCTION BYTES_TO_DWORD : DWORD
+
+VAR_INPUT
+	IN_00 : BYTE;    // BYTE 00
+	IN_01 : BYTE;    // BYTE 01
+	IN_02 : BYTE;    // BYTE 02
+	IN_03 : BYTE;    // BYTE 03
+END_VAR
+
+(* this Function combines the 2 BYTES to a word *)
+BYTES_TO_DWORD := DWORD_BYTE_ZZ;
+BYTES_TO_DWORD := BYTES_TO_DWORD OR SHL(BYTE_TO_DWORD(IN_03), SHFT_BYTE_03);
```

Comment:
The implementation can be simplified by using the bytewise access operator
e.g. `BYTES_TO_DWORD.%Bx` (or if thats not working on return values, a temporary variable)